src/components/calcite-tabs/calcite-tabs.stories.ts

```diff
@@ -28,7 +28,13 @@ export const Simple = (): string => html`
       <calcite-tab-title>Tab 4 Title</calcite-tab-title>
     </calcite-tab-nav>
 
-    <calcite-tab active><p>Tab 1 Content</p></calcite-tab>
+    <calcite-tab active>
```

Comment:
@caripizza @eriklharper how would you suggest we make it clear what each story covers visually? Would a simple comment suffice here or do we need a separate test? A separate one seems clearer to me, but may be overkill.

Reply:
Good question.  I'd say that if we were to revamp the stories for this component with the steps utilities, we could simply use the snapshot name to label what each story is visually testing, so for this particular change we would name the snapshot something like "horizontal overflow scroll" or something, but doing so will necessitate the small bit of refactoring to turn the Simple story into a stepStory with that additional screenshot in there.  We could make the horizontal overflow scroll snapshot show up in Tab 2 and just add a step that clicks Tab 2 and then takes a snapshot of the resulting UI to capture it under that name.

Reply:
Might be out of scope for this particular PR, but if @y0n4 wants to try and tackle that and learn the new steps utils at the same time, I'm totally available and glad to help her out on that!